<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extractor</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Progress informations</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="LogView">
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::NoWrap</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="FWCombo">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DownloadButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Download</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="HelpButton">
         <property name="text">
          <string>Help</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="AnalyzeSourceButton">
         <property name="text">
          <string>Analyze source</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BrowseButton">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="QuitButton">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
